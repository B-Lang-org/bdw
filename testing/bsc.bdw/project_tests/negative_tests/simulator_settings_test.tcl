
# Sets the simulation results location directory without opened project
handlers::set_sim_results_location mult4

# Opens the specified project
handlers::open_project project.bspec

# Sets the simulation results location directory without argument
handlers::set_sim_results_location

# Sets the simulation results location directory with wrong argument
handlers::set_sim_results_location mult3

# Sets the simulation results location directory 
handlers::set_sim_results_location negative_tests.exp

# Sets the simulator to iverilog
handlers::set_verilog_simulator 

# Sets the simulator to iverilog
handlers::set_verilog_simulator simulator

# Sets the simulator to iverilog
handlers::set_verilog_simulator iverilog -path

# Sets the simulator to iverilog
handlers::set_verilog_simulator iverilog -paths ddd

# Sets the simulator to iverilog
handlers::set_verilog_simulator iverilog -path ddd

# Closes the project
handlers::close_project

# Exits the workstation
exit
