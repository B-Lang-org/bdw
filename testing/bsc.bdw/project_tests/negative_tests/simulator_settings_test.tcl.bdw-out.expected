There is no open project
The simulation results directory is not specified
The file mult3 is not a directory.
The file negative_tests.exp is not a directory.
The simulator is not specified
Current version does not support the simulator 'simulator'
Argument -path expects an argument none found
Unknown option: -paths
The file ddd doesn't exist.
